src/dojos/cd-dojo-details.vue

```diff
@@ -44,6 +44,10 @@
         <div class="cd-dojo-details__heading">{{ $t('Upcoming Events') }}</div>
         <events-list v-if="dojoDetails.id" v-bind:dojo="dojoDetails"></events-list>
         <div class="cd-dojo-details__heading">{{ $t('Details') }}</div>
+        <a v-if="dojoDetails.geoPoint" :href="googleMapsLink" target="_blank">
+          <static-map google-api-key="AIzaSyC3xF9XV91bS2R14Gjmx3UQaKbGgAfHbE4" :zoom="15" :markers="googleMapsMarker" :paths="googleMapsPath" scale="2"
```

Comment:
key should be exposed as a global or through babel (see how GA is handled) not hardcoded